terraform/modules/paas/main.tf

```diff
@@ -34,6 +34,25 @@ resource cloudfoundry_app web_app {
   }
 }
 
+
+resource cloudfoundry_app worker_app {
```

Comment:
Please define and use new specific worker variables. eg: `web_app_instances` -> `worker_app_instances`
They can initially have the same value but we need to be able to scale them differently.

Reply:
done...